docarray/typing/url/text_url.py

```diff
@@ -13,6 +13,32 @@ class TextUrl(AnyUrl):
     Can be remote (web) URL, or a local file path.
     """
 
+    @classmethod
+    def mime_type(cls) -> str:
+        return 'text'
+
+    @classmethod
+    def extra_extensions(cls) -> List[str]:
+        """
+        List of extra file extensions for this type of URL (outside the scope of mimetype library).
+        """
+        return ['.md']
```

Comment:
avoid magic numbers, have all these `strings` in some place with meaningful constant names